packages/devtools_app/lib/src/ui/hover.dart

```diff
@@ -215,23 +223,93 @@ class HoverCard {
   }
 }
 
+/// Ensures that only one [HoverCard] is ever displayed at a time.
+class HoverCardTooltipController {
```

Comment:
should this be 'HoverCartController' instead of HoverCardTooltipController? adding 'tooltip' kind of makes it sound like this controls tooltips on the hover cards, not the cards themselves